slim/datasets/dataset_utils.py

```diff
@@ -53,15 +53,18 @@ def bytes_feature(values):
   return tf.train.Feature(bytes_list=tf.train.BytesList(value=[values]))
 
 
-def image_to_tfexample(image_data, image_format, height, width, class_id):
-  return tf.train.Example(features=tf.train.Features(feature={
-      'image/encoded': bytes_feature(image_data),
-      'image/format': bytes_feature(image_format),
-      'image/class/label': int64_feature(class_id),
-      'image/height': int64_feature(height),
-      'image/width': int64_feature(width),
-  }))
-
+def image_to_tfexample(image_data, image_format, height, width, class_id,
+                       coarse_class_id = None):
+  features = {
+    'image/encoded': bytes_feature(image_data),
+    'image/format': bytes_feature(image_format),
+    'image/class/label': int64_feature(class_id),
+    'image/height': int64_feature(height),
+    'image/width': int64_feature(width),
+  }
+  if coarse_class_id is not None:
+      features['image/class/coarse_label'] = int64_feature(coarse_class_id)
+  return tf.train.Example(features=tf.train.Features(feature=features))
```

Comment:
It's debatable whether we want to include the coarse_label as an extra feature: we could instead have a dict mapping from label to coarse_label.  Wdyt?